### DISCUSSION TITLE
Set-PnPList: Enable "Audience targeting settings" possible?

### DISCUSSION THREAD

**Discussion by SPRubi** [pnp/powershell#692] [Ideas]
Dear community, 

having the issue, that I don't get the checkbox acitvated for "enable audience targeting" in list settings. 
Can add the target audience field to site or list, add to content types etc. but don't get the setting activated. 
This prevents the field from being displayed in the "page properties" of a news. 
Could someone please add such function to the set-pnplist command?

Thanks and best regards,
Rubi

**Comment by murchelon** [comment]
This would be a very nice feature and would be in sync with the pnp 202209 provisioning template.
thanks a lot
